src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -38,8 +38,9 @@
 
 class LBWSGRiskEffect(LBWSGRiskEffect_):
     """Subclass of LBWSGRiskEffect to be compatible with the wide state table, meaning it
-    will query on child lookup columns. This also exposes the PAF to a pipeline so it is
-    accessible by other components in the model."""
+    will query on child lookup columns. This also exposes the PAF as a pipeline so it is
+    accessible by the neonatal causes component. The ACMR PAF will be used to calculate a
+    normalizing Z constant to modify CSMR pipelines for neonatal causes."""
```

Comment:
What is a z constant? I feel like saying just normalizing constant would be clearer if that's all it is.